overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -36703,7 +36703,7 @@
           "repeat_relative:1": 1,
           "ismain:1": 0,
           "name:8": "Electrolyzer",
-          "desc:8": "The Electrolyzer is an optional machine, but it\u0027s the easiest way to get Chlorine using only Water and Salt as the inputs."
+          "desc:8": "The Electrolyzer is an optional machine, but it is one of the ways to get Chlorine using only Water and Salt as the inputs. The other way is to just dissolve mekanism salt in a chemical dissolver."
```

Comment:
while this is by no means standard, machine and item names tend to be capitalized - `Mekanism Salt in a Chemical Dissolver`